docs/querying/aggregations.md

```diff
@@ -422,6 +389,121 @@ It is not possible to determine a priori how well this aggregator will behave fo
 
 For these reasons, we have deprecated this aggregator and recommend using the DataSketches Quantiles aggregator instead for new and existing use cases, although we will continue to support Approximate Histogram for backwards compatibility.
 
+
+## Expression aggregators
+
+### Expression aggregator
+
+Query time only aggregator that can aggregate results using [Druid expressions](./math-expr.md) functions to facilitate building custom functions.
```

Comment:
I think it would be helpful to show the expression aggregator here. Similar to what you have for the JavaScript aggregator on line 478.

Reply:
i don't really understand, there are several real examples of the expression aggregator after the table, and the table seems a more suitable place to explain the syntax rather than a pseudo json example like the javascript aggregator uses...

Reply:
I think it would probably be better if _all_ of the aggregators on this page had a table syntax form and then saved the json for realistic examples, but .. i wasn't very motivated to fix the whole page 😅

Reply:
Works for me. Fixing the whole page is quite an undertaking :)

Reply:
eh, i think maybe i will just try to fix the whole page once i get around to fixing up this PR based on review, seems like it won't be _that_ bad